tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
   "compilerOptions": {
     "declaration": true,
-    "target": "ES2021",
+    "target": "es5",
```

Comment:
ES5 is super duper old, why?

Reply:
Using `ESM` was giving me some issues with react and `es5` was the only target with no errors. 
Not using `modules` now so i will update this, should use `esnext` to target the latest version?